weaviate/collection/classes/filters.py

```diff
@@ -92,59 +92,90 @@ class _FilterValue(_Filters):
 
 
 class Filter:
+    """Define a filter based on a property to be used when querying a collection.
```

Comment:
Propose to add:

"To combine multiple filters, you can use `&` or `|` operators for each `AND` or `OR` operations."